flake.nix

```diff
@@ -57,8 +56,16 @@
               haskellPackages.haskell-language-server
               pkgs.cachix
               pkgs.nixpkgs-fmt
+              pkgs.secp256k1
               pkgs.stylish-haskell
               tasks
+
+              # https://nixos.wiki/wiki/FAQ/I_installed_a_library_but_my_compiler_is_not_finding_it._Why%3F
+              # Needed for binding secp256k1
+              pkgs.autoconf
+              pkgs.automake
+              pkgs.pkg-config
+              pkgs.libtool
```

Comment:
I'm not sure we need these ... maybe try the above change and see if they're still needed?